app/models/form_question.rb

```diff
@@ -8,17 +8,17 @@ class FormQuestion < ApplicationRecord
   after_commit :reindex_on_update, on: [:update]
 
   def reindex
-    UpdateIndexJob.perform_later('question', ESQuestionSerializer.new(question).as_json) if previous_changes[:question_id]
-    UpdateIndexJob.perform_later('response_set', ESResponseSetSerializer.new(response_set).as_json) if response_set
+    UpdateIndexJob.perform_later('question', question) if previous_changes[:question_id]
+    UpdateIndexJob.perform_later('response_set', response_set) if response_set
   end
 
   def reindex_on_update
     # While question can't actually change, previous_changes[:question_id] will exist if this form question was just created
-    UpdateIndexJob.perform_later('question', ESQuestionSerializer.new(question).as_json) if previous_changes[:question_id]
+    UpdateIndexJob.perform_later('question', ESQuestionSerializer.new(question)) if previous_changes[:question_id]
```

Comment:
This probably shouldn't be using a serializer anymore, correct? Also is there any performance to be gained by passing in just the id instead of the whole object (even if not running through the serializer first)?